include/extractor/segment_data_container.hpp

```diff
@@ -22,24 +24,24 @@ class CompressedEdgeContainer;
 
 namespace detail
 {
-template <bool UseShareMemory> class SegmentDataContainerImpl;
+template <osrm::storage::Ownership Ownership> class SegmentDataContainerImpl;
```

Comment:
Ah missed that one: You can omit the `osrm::` for all of these, since we are always in that namespace.